packages/utils/telemetry-utils/package.json

```diff
@@ -94,6 +94,11 @@
   },
   "typeValidation": {
     "version": "1.1.0",
-    "broken": {}
+    "broken": {
+      "RemovedFunctionDeclaration_originatedAsExternalError": {
+        "forwardCompat": false,
+        "backCompat": false
+      }
```

Comment:
Just so I can understand, should this be done on next or would it be ok in main ?